src/main/frontend/src/components/shoppage/ShopCategoryItem.js

```diff
@@ -50,10 +29,27 @@ function ShopCategoryItem({
             onSuperCategoryChange(category.categoryId);
           }}
         >
-          {isCollapsed ? "-" : "+"}
+          {!isCollapsed ? "-" : "+"}
```

Comment:
When writing if else blocks, positive and expected case should always be first. `isCollapsed ? '+' : '-'`